CMakeLists.txt

```diff
@@ -67,10 +34,7 @@ target_sources(N64Recomp PRIVATE
     ${CMAKE_SOURCE_DIR}/src/recompilation.cpp)
 
 target_include_directories(N64Recomp PRIVATE
-    "${CMAKE_SOURCE_DIR}/lib/rabbitizer/include"
-    "${CMAKE_SOURCE_DIR}/lib/rabbitizer/cplusplus/include"
     "${CMAKE_SOURCE_DIR}/lib/ELFIO"
-    "${CMAKE_SOURCE_DIR}/lib/fmt/include"
```

Comment:
fmt already makes the include folder public headers so we don't have to include them manually